SPECS/csi-driver-lvm/csi-driver-lvm.spec

```diff
@@ -0,0 +1,41 @@
+Summary:        Container storage interface for logical volume management
+Name:           csi-driver-lvm
+Version:        0.4.1
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/metal-stack/csi-driver-lvm
+#Source0:       https://github.com/metal-stack/%{name}/archive/refs/tags/v%{version}.tar.gz
+Source0:        %{name}-%{version}-govendorcached.tar.gz
+# Below is a manually created tarball, no download link.
+# We're using pre-populated GO dependencies from this tarball, since network is disabled during build time.
+#   1. wget https://github.com/metal-stack/%{name}/archive/refs/tags/v%{version}.tar.gz -o %%{name}-%%{version}.tar.gz
+#   2. tar -xf %%{name}-%%{version}.tar.gz
+#   3. cd %%{name}-%%{version}
+#   4. go mod vendor
+#   5. tar  --sort=name \
+#           --mtime="2022-09-18 00:00Z" \
+#           --owner=0 --group=0 --numeric-owner \
+#           --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \
+#           -cf %%{name}-%%{version}-govendorcached.tar.gz .
+BuildRequires:  golang
+
+%description
+csi-driver-lvm utilizes local storage of Kubernetes nodes to provide persistent storage for pods
+
+%prep
+%setup -q
+
+%build
+make %{?_smp_mflags}
+
```

Comment:
Are there any project tests we could add and run inside a `%check` section? It would be great to have them here, so we catch regressions and know the project works correctly under Mariner.

Reply:
We don't have testing here, but when the customized image get created, we have a sanity test to do the testing work. Also, when leveraging the helm-chart, a testing will be run as well.

Reply:
Closed.